code/datums/jobs/job/shipside.dm

```diff
@@ -804,17 +804,17 @@ Make sure that the doctors and nurses are doing their jobs and keeping the marin
 	glasses = /obj/item/clothing/glasses/hud/health
 	mask = /obj/item/clothing/mask/surgical
 	head = /obj/item/clothing/head/cmo
-	r_store = /obj/item/storage/pouch/medkit/medic
-	l_store = /obj/item/storage/pouch/surgery
+	r_store = /obj/item/storage/pouch/surgery
+	l_store = /obj/item/storage/pouch/medkit/medic
```

Comment:
Why swap the sides?

Reply:
Consistency. Also I need to do work on this, I'll get around to it within the next couple days.